unit-tests/src/test/java/org/eclipse/collections/impl/AbstractRichIterableTestCase.java

```diff
@@ -1504,7 +1502,7 @@ public void chunk_zero_throws()
     @Test
     public void chunk_large_size()
     {
-        RichIterable<String> collection = FastList.newListWith("1", "2", "3", "4", "5", "6", "7");
+        RichIterable<String> collection = this.newWith("1", "2", "3", "4", "5", "6", "7");
         Assert.assertEquals(collection, collection.chunk(10).getFirst());
```

Comment:
Since you're changing this method, it would be nice to change this line to use getOnly() instead of deprecated getFirst() method. Same case for the overridden methods in child classes.